android/app/src/main/java/org/uwaterloo/subletr/pages/home/HomePageViewModel.kt

```diff
@@ -23,4 +27,11 @@ class HomePageViewModel @Inject constructor(
 	override val uiStateStream: Observable<HomePageUiState> = homeListChildViewModel.uiStateStream.map {
 		it
 	}
+
+	init {
+		viewModelScope.launch {
+			val loc = locationService.getLocation()
+			println(loc.toString())
+		}
+	}
```

Comment:
@leeminkyu16 wanna check it out?

Reply:
Yup, it'll still crash sometimes even if you do this.
Also, `viewModelScope.launch` just ensures that the coroutine is properly disposed if you destroy the `viewModel`.
AFAIK, it won't actually fix anything (although it might reduce the crash rate?).

Reply:
hmm I wasn't getting crashes when I was running it, I'll look at it again

Reply:
I do find that it's really random lol
Like it hasn't crashed for me at all today and I've run it multiple times, even on startup